src/Sulu/Bundle/MediaBundle/DataFixtures/ORM/LoadCollectionTypes.php

```diff
@@ -24,43 +24,28 @@ class LoadCollectionTypes extends AbstractFixture implements OrderedFixtureInter
      */
     public function load(ObjectManager $manager)
     {
-        // force id = 1
+        // set id manually
         $metadata = $manager->getClassMetaData(CollectionType::class);
         $metadata->setIdGeneratorType(\Doctrine\ORM\Mapping\ClassMetadata::GENERATOR_TYPE_NONE);
 
-        $defaultCollectionType = $this->createCollectionType(1, 'collection.default', 'Default');
-        $manager->persist($defaultCollectionType);
+        // create or update collectiontype with id 1
+        $defaultCollectionType = new CollectionType();
+        $defaultCollectionType->setId(1);
+        $defaultCollectionType = $manager->merge($defaultCollectionType);
```

Comment:
This order is needed to ensure the desired behaviour;

- We create a **unmanaged** CollectionType object and set the required id.
- The merge-call returns a **managed** copy of the CollectionType object which is merged with an eventually stored CollectionType object of the database.
- Now that we got a managed CollectionType object, we can update the respective properties and the changes will get saved to the database on flush.

If the merge-method is called after the setters of the entity, the setters are called on an unmanaged (detached) entity and will not be saved on flush.